Sources/Orbit/Components/Stepper.swift

```diff
@@ -0,0 +1,262 @@
+import SwiftUI
+
+/// Enables incremental changes of a counter without a direct input.
+///
+/// - Note: [Orbit definition](https://orbit.kiwi/components/interaction/stepper/)
+public struct Stepper: View {
+
+    @Binding var value: Int
+
+    let minValue: Int
+    let maxValue: Int
+    let isEnabled: Bool
+    let style: Style
+
+    @Environment(\.colorScheme) var colorScheme
+    
+    @State var isIncreased = false
+
+    public var body: some View {
+        HStack(alignment: .center, spacing: 0) {
+            decrementButton
+                .accessibility(addTraits: [.isButton])
+
+            valueContainer
+                .frame(minWidth: 40)
+                .accessibility(value: .init("\(value)"))
+
+            incrementButton
+                .accessibility(addTraits: [.isButton])
+        }
+    }
+
+    @ViewBuilder var valueContainer: some View {
+        Text("\(value)")
+            .foregroundColor(colorScheme == .light ? .cloudDark : .cloudLight)
+            .transition(
+                AnyTransition.asymmetric(
+                    insertion: .move(edge: isIncreased ? .top : .bottom),
+                    removal: .move(edge: isIncreased ? .bottom : .top)
+                )
+                .combined(with: .opacity)
+            )
+            .id("\(value)")
+    }
+
+    @ViewBuilder var decrementButton: StepperButton {
+        StepperButton(
+            isIncrement: false,
+            isEnabled: isEnabled ? value > minValue : false,
+            style: style,
+            clickAction: {
+                isIncreased = false
+
+                withAnimation {
+                    value -= 1
+                }
+            }
+        )
+    }
+
+    @ViewBuilder var incrementButton: StepperButton {
+        StepperButton(
+            isIncrement: true,
+            isEnabled: isEnabled ? value < maxValue : false,
+            style: style,
+            clickAction: {
+                isIncreased = true
+
+                withAnimation {
+                    value += 1
+                }
+            }
+        )
+    }
+
+    /// Creates Orbit Stepper component.
+    public init(
+        value: Binding<Int>,
+        minValue: Int,
+        maxValue: Int,
+        isEnabled: Bool = true,
+        style: Style
+    ) {
+        self._value = value
+        self.minValue = minValue
+        self.maxValue = maxValue
+        self.isEnabled = isEnabled
+        self.style = style
+    }
+}
+
+// MARK: - Subtypes
+extension Stepper {
+
+    struct StepperButton: View {
+
+        let isIncrement: Bool
+        let isEnabled: Bool
+        let style: Stepper.Style
+        let clickAction: () -> Void
+
+        var body: some View {
+            SwiftUI.Button(action: clickAction) {
+                ZStack {
+                    Circle()
+                        .fill(isEnabled ? style.backgroundActiveColor : style.backgroundColor)
+                        .frame(width: 24, height: 24)
```

Comment:
Sizes should not be fixed, but driven by text sizes. Example: Badge or BadgeList icons

Reply:
Revised